homeassistant/components/blueprint/models.py

```diff
@@ -127,9 +132,33 @@ def inputs(self):
         """Return the inputs."""
         return self.config_with_inputs[CONF_USE_BLUEPRINT][CONF_INPUT]
 
+    @property
+    def inputs_with_default(self):
+        """Return the inputs and fallback to defaults."""
+        no_input = self.blueprint.placeholders - set(self.inputs)
+
+        inputs_with_default = dict(self.inputs)
+
+        for input in no_input:
+            blueprint_input = self.blueprint.inputs[input]
+            if isinstance(blueprint_input, dict):
+                inputs_with_default[input] = blueprint_input.get("default")
+
+        return inputs_with_default
+
     def validate(self) -> None:
         """Validate the inputs."""
-        missing = self.blueprint.placeholders - set(self.inputs)
+        no_input = self.blueprint.placeholders - set(self.inputs)
```

Comment:
We should be able to use this:
```suggestion
        no_input = self.blueprint.placeholders - set(self.inputs_with_default)
```

Reply:
And then drop the new code.